test/BaseTokenWrapper.t.sol

```diff
@@ -678,6 +682,7 @@ abstract contract BaseTokenWrapperTest is Test {
 
   function testFuzzSupplyToken(uint256 amount, address referee) public {
     amount = bound(amount, 1, MAX_DEAL_AMOUNT);
+    vm.assume(IAToken(aTokenOut).balanceOf(referee) == 0);
```

Comment:
why is this needed? tests start with a fresh state so we can assume they have 0 balance

Reply:
The parent tests fork mainnet, so this causes some of those to fail

Reply:
We could deal referee 0 instead if that's cleaner

Reply:
ah, that's probably better if works well, because of fuzzing engine overhead

Reply:
Interesting... I actually get an issue when I do the deal. I suppose there actually is more overhead in setting the balance to 0 via deal than just checking that the balance is 0

```[FAIL. Reason: revert: stdStorage find(StdStorage): Slot(s) not found.; counterexample: calldata=0x1d05d98a0000000000000000000000000000000000000000000000000000000000002b49000000000000000000000000000000000000000000000000000000000000c938 args=[11081 [1.108e4], 0x000000000000000000000000000000000000c938]] testFuzzSupplyToken(uint256,address) (runs: 0, μ: 0, ~: 0)```

Should I just leave as is?